src/legacy/core_plugins/data/public/stubs.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
Move to NP `src/plugins/data/public/index_patterns`.

Reply:
Moved